mailscanner/user_manager.php

```diff
@@ -372,7 +372,7 @@ function change_state(id, filter) {
         echo ' <tr><td class="heading">' . __('pontspam12') . '</td><td><input type="text" name="spamscore" value="' . $row->spamscore . '" size="4"> <span style="font-size:90%">0=' . __('usedefault12') . '</span></td></tr>'."\n";
         echo ' <tr><td class="heading">' . __('hpontspam12') . '</td><td><input type="text" name="highspamscore" value="' . $row->highspamscore . '" size="4"> <span style="font-size:90%">0=' . __('usedefault12') . '</span></td></tr>'."\n";
         echo '<tr><td class="heading">' . __('action_0212') . '</td><td><input type="reset" value="' . __('reset12') . '">&nbsp;&nbsp;<input type="submit" value="' . __('update12') . '"></td></tr>'."\n";
-        echo '</table></form><br>'."\n";
+        echo '</table></form><script>$(\'form\').attr(\'autocomplete\',\'off\');</script><br>'."\n";
```

Comment:
I don't have JQuery in the server!

And it works?

I've found this here: https://scotsscripts.com/blog/html-css-trick-how-to-turn-off-auto-complete.html